BinaryEncoding.md

```diff
@@ -1,5 +1,7 @@
 # Binary Encoding
 
+## Note: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/core/binary/index.html).
```

Comment:
Nit: "here" is not a great link text; better to link "normative documentation". LGTM otherwise.

Reply:
Done